v0/sources/canny/voterMapping.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "sourceKeys": "created",
+    "destKeys": ["timestamp", "originalTimestamp"]
+  },
+  {
+    "sourceKeys": "object",
+    "destKeys": "properties",
+    "metadata": {
+      "excludes": ["voter"]
+    }
```

Comment:
We are not using metadata in `setPropertiesV2` function. So it is not needed.

Reply:
Ok, removed it